docs/operations/data_migration.md

```diff
@@ -586,3 +586,47 @@ Example: `cf ssh getgov-za`
 |   | Parameter                  | Description                                                                 |
 |:-:|:-------------------------- |:----------------------------------------------------------------------------|
 | 1 | **debug**                  | Increases logging detail. Defaults to False.                                |
+
+
+## Populate First Ready
+This section outlines how to run the `populate_organization_type` script.
```

Comment:
Add a max two second description describing what this does, so a future dev would know